docs/tutorials/word-tutorial.md

```diff
@@ -0,0 +1,962 @@
+    docBody.insertParagraph("Office has several versions, including Office 2016, Office 365 Click-to-Run, and Office Online.",
+                            "Start");
+    ```
+
+### Test the add-in
+
+1. Open a Git bash window, or Node.JS-enabled system prompt, and navigate to the **Start** folder of the project.
+
+2. Run the command `npm run build` to transpile your ES6 source code to an earlier version of JavaScript that is supported by all the hosts where Office Add-ins can run.
+
+3. Run the command `npm start` to start a web server running on localhost.
+
+4. Sideload the add-in by using one of the following methods:
+
+    - Windows: [Sideload Office Add-ins on Windows](../testing/create-a-network-shared-folder-catalog-for-task-pane-and-content-add-ins.md)
+
+    - Word Online: [Sideload Office Add-ins in Office Online](../testing/sideload-office-add-ins-for-testing.md#sideload-an-office-add-in-in-office-online)
+
+    - iPad and Mac: [Sideload Office Add-ins on iPad and Mac](../testing/sideload-an-office-add-in-on-ipad-and-mac.md)
+
+5. On the **Home** menu of Word, select **Show Taskpane**.
+
+6. In the task pane, choose **Insert Paragraph**.
+
+7. Make a change in the paragraph.
+
+8. Choose **Insert Paragraph** again. Note that the new paragraph is above the previous one because the `insertParagraph` method is inserting at the "start" of the document's body.
+
+    ![Word tutorial - Insert Paragraph](../images/word-tutorial-insert-paragraph.png)
+
+## Format text
+
+In this step of the tutorial, you'll change the font of text, and use both built-in and custom styles on the text.
+
+### Apply a built-in style to text
+
+1. Open the project in your code editor. 
+
+2. Open the file index.html.
+
+3. Just below the `div` that contains the `insert-paragraph` button, add the following markup:
+
+    ```html
+    <div class="padding">            
+        <button class="ms-Button" id="apply-style">Apply Style</button>            
+    </div>
+    ```
+
+4. Open the app.js file.
+
+5. Just below the line that assigns a click handler to the `insert-paragraph` button, add the following code:
+
+    ```js
+    $('#apply-style').click(applyStyle);
+    ```
+
+6. Just below the `insertParagraph` function, add the following function:
+
+    ```js
+    function applyStyle() {
+        Word.run(function (context) {
+            
+            // TODO1: Queue commands to style text.
+
+            return context.sync();
+        })
+        .catch(function (error) {
+            console.log("Error: " + error);
+            if (error instanceof OfficeExtension.Error) {
+                console.log("Debug info: " + JSON.stringify(error.debugInfo));
+            }
+        });
+    }
+    ``` 
+
+7. Replace `TODO1` with the following code. Note that the code applies a style to a paragraph, but styles can also be applied to ranges of text.
+
+    ```js
+    const firstParagraph = context.document.body.paragraphs.getFirst();
```

Comment:
There's several instances of `const` in the tutorial JavaScript. I believe that's acceptable as of ES6, but my understanding is we're sticking to pretty vanilla JS. Should these all be `var` instead?

Reply:
Good catch; I'll change these to `var` instead.